compose/app/src/main/java/com/chartboost/mediation/sdk/demo/AdController.kt

```diff
@@ -0,0 +1,501 @@
+package com.chartboost.mediation.sdk.demo
```

Comment:
did we need copyright headers for the demo app?

Reply:
the existing XML app doesn't have any so I'll keep it like that